src/libs/export.c

```diff
@@ -1303,6 +1303,32 @@ void gui_init(dt_lib_module_t *self)
   dt_bauhaus_combobox_add(d->intent, _("absolute colorimetric"));
   gtk_box_pack_start(GTK_BOX(self->widget), d->intent, FALSE, TRUE, 0);
 
+  tooltip = g_strdup_printf(_("• perceptual : "
+                              "smoothly moves out-of-gamut colors into gamut,"
+                              "preserving gradations, but distorts in-gamut colors in the process."
+                              "\n\n"
+
+                              "• relative colorimetric : "
+                              "reproduces colors with as little desaturation as possible."
+                              " out of gamut colors are moved to the nearest point within gamut."
+                              " colors are all scaled to the white point of the destination color"
+                              " space."
+                              "\n\n"
+
+                              "• saturation : "
+                              "designed to present eye-catching business graphics"
+                              " by preserving the saturation. (not suited for photography)."
+                              "\n\n"
+
+                              "• absolute colorimetric : "
+                              "reproduces colors scaled to the white point of the source color"
```

Comment:
I would phrase it "adapt white point of the image to the white point of the destination medium and do nothing else"

Reply:
Works for me.